app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,29 @@
+
+<div class="container w-50 mx-auto shadow p-3 mb-5 bg-white rounded">
+  <h2>Log in form</h2>
```

Comment:
- [ ] Kindly make sure that the design of the login and signup page matches the wireframes from the design guidelines.
- You can take a look at the wireframes in the [sneak peek](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md) of the project.